addons/account/views/report_invoice.xml

```diff
@@ -19,30 +19,30 @@
                         <span t-field="o.number"/>
                     </h2>
 
-                    <div class="row mt32 mb32">
-                        <div class="col-auto" t-if="o.name" name="description">
-                            <strong>Description:</strong>
-                            <p t-field="o.name"/>
+                    <div id="informations" class="row mt32 mb32">
+                        <div class="col-auto mt-2 mb-2" t-if="o.name" name="description">
+                            <strong>Description:</strong><br/>
+                            <span t-field="o.name"/>
```

Comment:
@JKE-be can I change this tag in v12 or not?

Reply:
we try to avoid !